README.md

```diff
@@ -1,18 +1,1481 @@
+  - `relayState` - Optional state value that is persisted for the lifetime of the recovery transaction
+
+```javascript
+authClient.unlockAccount({
+  username: 'dade.murphy@example.com',
+  factorType: 'SMS', // 'EMAIL' or undefined for trusted applications
+})
+.then(function(transaction) {
+  // if factorType is undefined
+  // return transaction.recovery({
+  //   recoveryToken: '00xdqXOE5qDZX8-PBR1bYv8AESqIFinDy3yul01tyh'
+  // });
+
+  return transaction.verify({
+    passCode: '123456' // The passCode from the SMS
+  });
+})
+.then(function(transaction) {
+  if (transaction.status === 'SUCCESS') {
+    authClient.session.setCookieAndRedirect(transaction.sessionToken);
+  } else {
+    throw 'We cannot handle the ' + transaction.status + ' status';
+  }
+})
+.fail(function(err) {
+  console.error(err);
+});
+```
+
+## verifyRecoveryToken(options)
+
+Validates a recovery token that was distributed to the end-user to continue the recovery transaction.
+
+- `recoveryToken` - Recovery token that was distributed to end-user via out-of-band mechanism such as email
+
+```javascript
+authClient.verifyRecoveryToken({
+  recoveryToken: '00xdqXOE5qDZX8-PBR1bYv8AESqIFinDy3yul01tyh'
+})
+.then(function(transaction) {
+  if (transaction.status === 'SUCCESS') {
+    authClient.session.setCookieAndRedirect(transaction.sessionToken);
+  } else {
+    throw 'We cannot handle the ' + transaction.status + ' status';
+  }
+})
+.fail(function(err) {
+  console.error(err);
+});
+```
+
+## tx.resume()
+
+Resumes an in-progress transaction. This is useful if a user navigates away from the login page before authentication is complete.
+
+```javascript
+authClient.tx.exists()
+.then(function(exists) {
+  if (exists) {
+    return authClient.tx.resume();
+  }
+  throw new Error('a session does not exist');
+})
+.then(function(transaction) {
+  console.log('current status:', transaction.status);
+})
+.fail(function(err) {
+  console.error(err);
+});
+```
+
+## tx.exists()
+
+Check for a transaction to be resumed. This is synchronous and returns `true` or `false`.
+
+```javascript
+authClient.tx.exists()
+.then(function(exists) {
+  if (exists) {
+    console.log('a session exists');
+  } else {
+    console.log('a session does not exist');
+  }
+})
+.fail(function(err) {
+  console.error(err);
+});
+```
+
+## transaction.status
+
+When Auth Client methods resolve, they return an object that encapsulates the new state in the authentication flow, a **transaction** object. This object contains metadata about the current state, and methods that can be used to progress to the next state.
+
+Below are sample transaction objects and usage of their methods:
+
+### LOCKED_OUT
+
+The user account is locked; self-service unlock or admin unlock is required.
+
+```javascript
+{
+  status: 'LOCKED_OUT',
+  unlock: function(options) { /* returns another transaction */ },
+  cancel: function() { /* terminates the auth flow */ },
```

Comment:
Should we also be documenting the cancel flow here as well? Since it's shared amongst a lot of methods, maybe we should have a section here, but link to another section that has shared transaction methods.